### DISCUSSION TITLE
test if price is 0.00

### DISCUSSION THREAD

**Discussion by rogerg1967** [openstore-ecommerce/OpenStore#250] [Q&A]
I am trying to check if a price is 0.00 and if it is replaced with the text Free Gift.
Is there something like 
@if (product.PriceUnit > 0) ...
I cant find anything with the correct test.
Thanks Roger

**Comment by DNNMonster** [comment]
I think you're looking for product.FromPrice() or product.BestPrice()

**Reply by rogerg1967** [reply]
Yes that worked thanks.
